### PR TITLE
Bug fixes for Policy Managed Indices and Indexes pages

### PR DESCRIPTION
**Pull request by tandonks**
### Description
This PR achieves 3 things:
1) Bug fixes in Policy Managed Indices action button
2) Reverts yarn.lock file changes that went with the previos PR: https://github.com/opensearch-project/index-management-dashboards-plugin/pull/1124
3) Fixes bug in indexes page

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx [hunk 1]
```diff
@@ -561,18 +541,24 @@ export class ManagedIndices extends MDSEnabledComponent<ManagedIndicesProps, Man
...
+              onShow(RolloverAliasModal, {
+                index: selectedItems[0].index,
+                core: this.context,
+              })
+            }
+          >
+            Edit rollover alias
+          </EuiContextMenuItem>
+        )}
+      </ModalConsumer>,
```
Comment (SuZhou-Joe) [inline public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx:561]: Any reason we are changing the logic here?

File: public/pages/ManagedIndices/components/InfoModal/InfoModal.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ import {
   EuiOverlayMask,
   EuiCodeBlock,
 } from "@elastic/eui";
+import { size } from "lodash";
```
Comment (SuZhou-Joe) [inline public/pages/ManagedIndices/components/InfoModal/InfoModal.tsx:17]:
```suggestion
```

File: public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx (hunk 1 above)
Comment (tandonks) [reply public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx:561]: There was a bug in that implementation there due to which the button was not working properly, so have fixed that

File: public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx [hunk 3]
```diff
@@ -435,7 +435,7 @@ export class ManagedIndices extends MDSEnabledComponent<ManagedIndicesProps, Man
       showEditModal,
       isPopoverOpen,
     } = this.state;
-
+    console.log(this.context);
```
[the remark this answers is not in the document]
Comment (tandonks) [reply public/pages/ManagedIndices/containers/ManagedIndices/ManagedIndices.tsx:438]: Done

**Comment by CaptainDredge** [comment]
@tandonks closing this PR since you've opened a new one with required changes

**Pull request closed without merging**